packages/preview-middleware/README.md

```diff
@@ -185,3 +185,35 @@ return flp.router
 - `rootProject` - [Reader](https://sap.github.io/ui5-tooling/stable/api/@ui5_fs_AbstractReader.html) to access resources of the root project
 - `middlewareUtil` - [MiddlewareUtil](https://sap.github.io/ui5-tooling/v3/api/@ui5_server_middleware_MiddlewareUtil.html) of the UI5 server
 - `logger` - Logger instance for use in the middleware.
```

Comment:
```suggestion
- `logger` - Logger instance to be used in the middleware.
```

Reply:
Done in f41acea6cdc7bcf341fd9c72b7a4306e0012a3b5